deploy/helm/sumologic/values.yaml

```diff
@@ -3977,7 +3977,7 @@ metadata:
         ## ref: https://github.com/SumoLogic/opentelemetry-collector/blob/main/processor/batchprocessor
         batch:
           ## Number of spans after which a batch will be sent regardless of time
-          send_batch_size: 16_384
+          send_batch_size: 1_024
           ## Time duration after which a batch will be sent regardless of size
           timeout: 30s
```

Comment:
Should we update timeout as well?

Reply:
Yeah, a lower value like `1s` is better imo.

Reply:
I will update that